lib/inspec/cli.rb

```diff
@@ -168,9 +168,70 @@ def archive(path)
     pretty_handle_exception(e)
   end
 
-  desc 'exec PATHS', 'run all test files at the specified PATH.'
+  desc 'exec LOCATIONS', 'run all test files at the specified LOCATIONS.'
   long_desc <<~EOT
-    Loads the given profile(s) and fetches their dependencies if needed.  Then connects to the target and executes any controls contained in the profiles.  One or more reporters are used to generate output.  If all tests passed (no fails, no skips) exit code 0 is returned.  If some tests skipped but none failed, exit code 101 is returned. If at least one test failed, exit code 100 is returned.  If inspec failed for any other reason, exit code 1 is returned.
+    Loads the given profile(s) and fetches their dependencies if needed. Then
+    connects to the target and executes any controls contained in the profiles.
+    One or more reporters are used to generate output. If all tests passed
+    (no fails, no skips) exit code 0 is returned. If some tests skipped but
+    none failed, exit code 101 is returned. If at least one test failed, exit
+    code 100 is returned. If inspec failed for any other reason, exit code 1
+    is returned.
+
+    Below are some examples of using `exec` with different test LOCATIONS:
+
+    Automate:
+      ```
+      inspec compliance login
+      inspec exec compliance://username/linux-baseline
+      ```
+
+    Supermarket:
+      ```
+      inspec exec supermarket://username/linux-baseline
+      ```
+
+    Local profile (executes all tests in `controls/`):
+      ```
+      inspec exec /path/to/profile
+      ```
+
+    Local single test (doesn't allow attributes or custom resources)
+      ```
+      inspec exec /path/to/a_test.rb
+      ```
+
+    Git via SSH (uses SSH keys if repo is private):
+      ```
+      inspec exec git@github.com:dev-sec/linux-baseline.git
+      ```
+
+    Git via HTTPS (.git is required):
```

Comment:
Consider clarifying that "(the '.git' suffix is required)"

On first reading I thought you meant that the `git` executable was required, and the `.` was a typo.

Reply:
This is why we have multiple eyes looking! Will change. Thanks!